packages/services/document-service/src/main/java/software/uncharted/terarium/documentserver/responses/xdd/XDDAutocompleteResponse.java

```diff
@@ -0,0 +1,10 @@
+package software.uncharted.terarium.documentserver.responses.xdd;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+@Data
+@Accessors(chain = true)
+public class XDDAutocompleteResponse {
+	private Object suggest;
```

Comment:
As an example of what I mean here - take a look at `XDDArticlesResponseOk`

Reply:
It should be part of this PR

Reply:
I was thinking about that; how important is it? Because the object that is returned from that API is a really ugly nested structure of which the frontend is only concerned with one specific field. It seems like a lot of code bloat to add?

Reply:
I agree on just using Response then, and if we need to do some backend manipulation we can introduce it's own response type then.

Reply:
Seems like a reasonable solution to me.